go-controller/cmd/ovndbchecker/ovndbchecker.go

```diff
@@ -182,6 +182,10 @@ func runOvnKubeDBChecker(ctx *cli.Context) error {
 		return err
 	}
 
+	if err = ovndbmanager.EnableDBMemTrimming(); err != nil {
```

Comment:
this seems pretty new https://www.mail-archive.com/ovs-dev@openvswitch.org/msg48731.html

will not fail on old OVN versions?

Reply:
yes it will fail on older OVN versions, but in upstream ovn-k8s we only ever support latest right now (no concept of releases via tags or branches)